docs/index.rst

```diff
@@ -69,26 +69,45 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
 
 * `Plotting tutorial <http://nbviewer.jupyter.org/github/OceanParcels/parcels/blob/master/examples/tutorial_plotting.ipynb>`_ for further explanation on the plotting capabilities of Parcels
 
-Installing Parcels
-===================================
+Installing Parcels on Linux and macOS
+=====================================
+
+Parcels depends on a working Python installation, a netCDF installation, a C
+compiler, and various Python packages. 
+
+The simplest way to install Parcels is to use Anaconda and Conda-Forge for all 
+requirements. This is the “batteries-included” solution probably suitable for most users.
+
+1. Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/. Be sure to download the Python 2.7 version.
+
+2. Download Parcels' `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/install_via_conda/environment.yml>`_ file and run::
+
+    source $HOME/miniconda2/bin/activate root
+    conda env create -f environment.yml -n py2_parcels
+    source $HOME/miniconda2/bin/activate py2_parcels
+    pip install git+https://github.com/OceanParcels/parcels.git@master
+
+3. Validate the installation by downloading the `example_peninsula.py <https://raw.githubusercontent.com/OceanParcels/parcels/master/examples/example_peninsula.py>`_ file and then run::
+
+    python example_peninsula.py --fieldset 100 100
+
+4. Optionally, if you want to run all the examples and tutorials, you may want to download the example hydrodynamic fields by downloading the `pull_data.py <https://raw.githubusercontent.com/OceanParcels/parcels/master/parcels/scripts/pull_data.py>`_ file and then run::
+
+    python pull_data.py
 
-The latest version of Parcels, including tests and examples, 
-can be obtained directly from github via::
-    git clone https://github.com/OceanParcels/parcels.git
-    cd parcels; pip install -r requirements.txt
-    python scripts/pull_data.py
-    export PYTHONPATH="$PYTHONPATH:$PWD"
+5. The next time you log in to your machine and want to work with Parcels, activate the environment with::
 
-In order for Parcels to work from any directory, add the following line to 
-your ~/.bash_profile::
-    export PYTHONPATH="$PYTHONPATH:$PWD"
+    source $HOME/miniconda2/bin/activate py2_parcels
```

Comment:
@willirath is there a reason you suggested to `source $HOME/miniconda2/bin/activate py2_parcels` whereas in most conda documentation and tutorials people use `source activate py2_parcels`?

Reply:
The short version needs the Conda's `activate` executable in the Path.  During the Miniconda installation, the users have to decide if they want to add the installation to the Path. It is debatable whether this is a good idea or not.  (I prefer not to.)

The long version which explicitly includes the full path to `activate`, is working in any case.

Reply:
Ok with the source comment, but you changed here the meaning of the sentence.
It was already stated that user has to source the activate file (line 87).
The point here is to say that the source can be written in the .bash_profile

Reply:
Well, but do we really want people to source the Parcels environment in their `.bash_rc`? That goes against the idea of conda environments, where you only source them if you need them. So I prefer not to suggest adding to `.bash_rc`

Reply:
Ok, you're right